src/hapi_schema/db_refugees.py

```diff
@@ -0,0 +1,113 @@
+"""OperationalPresence table and view."""
+
+import enum
+from datetime import datetime
+
+from sqlalchemy import (
+    CheckConstraint,
+    DateTime,
+    Enum,
+    ForeignKey,
+    Integer,
+    select,
+)
+from sqlalchemy.orm import Mapped, aliased, mapped_column, relationship
+
+from hapi_schema.db_location import DBLocation
+from hapi_schema.utils.base import Base
+from hapi_schema.utils.shared_enums import Gender
+from hapi_schema.utils.view_params import ViewParams
+
+
+class PopulationGroup(enum.Enum):
+    REFUGEES = "refugees"
+    POC = "PoC"
+    NULL = "*"
+
+
+class DBRefugees(Base):
+    __tablename__ = "refugees"
+    __table_args__ = (
+        CheckConstraint(
+            "origin_location_ref != asylum_location_ref",
+            name="origin_location",
+        ),
```

Comment:
There are rows with internal refugees (same origin and destination).  They pass all other criteria.  Should I exclude these?

Reply:
Thanks for flagging this Briar, after discussion [here](https://humdata.slack.com/archives/C04UD1VPY2G/p1714992725343199), I've removed the constraint